From d9cae55a7b6a265275098b7b0612ec4e1023a967 Mon Sep 17 00:00:00 2001
From: Tianmi Chen <tianmi.chen@intel.com>
Date: Tue, 30 Jul 2013 13:36:28 +0800
Subject: Add HEVC/H.265 coding type (to the OMX_VIDEO_CODINGTYPE enum)

BZ: 120796

Add HEVC/H.265 coding type (to the OMX_VIDEO_CODINGTYPE enum)

Category: feature differentiation
Domain: Video.Media-Video Codecs
Origin: internal
Upstream-Candidate: no

Change-Id: Ia93f4cfa279f610ccadf8c85fc45d04eae820ca7
Orig-Change-Id: Ia8478b6212bea33e1ef52f6d3287481066f4a0fb
Signed-off-by: Tianmi Chen <tianmi.chen@intel.com>
Signed-off-by: Dan Liang <dan.liang@intel.com>
---
 include/media/openmax/OMX_Video.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/media/openmax/OMX_Video.h b/include/media/openmax/OMX_Video.h
index 4441a7a..17b4868 100644
--- a/include/media/openmax/OMX_Video.h
+++ b/include/media/openmax/OMX_Video.h
@@ -87,6 +87,7 @@ typedef enum OMX_VIDEO_CODINGTYPE {
     OMX_VIDEO_CodingMJPEG,      /**< Motion JPEG */
     OMX_VIDEO_CodingVP8,        /**< Google VP8, formerly known as On2 VP8 */
     OMX_VIDEO_CodingVP9,        /**< Google VP9 */
+    OMX_VIDEO_CodingHEVC,       /**< H.265/HEVC */
     OMX_VIDEO_CodingKhronosExtensions = 0x6F000000, /**< Reserved region for introducing Khronos Standard Extensions */ 
     OMX_VIDEO_CodingVendorStartUnused = 0x7F000000, /**< Reserved region for introducing Vendor Extensions */
     OMX_VIDEO_CodingMax = 0x7FFFFFFF
-- 
1.7.4.1

